app/workers/delete_mute_worker.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DeleteMuteWorker
+  include Sidekiq::Worker
+
+  def perform(mute_id)
+    mute = Mute.find_by(id: mute_id)
+    UnmuteService.new.call(mute.account, mute.target_account, mute) if mute
```

Comment:
here, I'd just do something like `UnmuteService.new.call(mute.account, mute.target_account) if mute && mute.expired?`.

Reply:
(About your four comments above) I added support for updating the duration value through the API. I also have to notice you that the feature of updating duration is not supported in Web UI 
operations at the moment...